AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -318,6 +325,7 @@ - (void)_populateFromEntireDataSourceWithMutableNodes:(NSMutableArray *)nodes mu
 
 - (void)beginUpdates
 {
+  [_editingTransactionQueue waitUntilAllOperationsAreFinished];
   // Begin queuing up edit calls that happen on the main thread.
   // This will prevent further operations from being scheduled on _editingTransactionQueue.
   // It's fine if there is an in-flight operation on _editingTransactionQueue,
```

Comment:
Update this comment / delete at least the lines referring to in-flight operations.  I assume the -wait call turned out to be necessary here (I remember adding it in my patch and believe it did help test behavior), and thus we are not actually incrementing it with operations in flight.